locallibrary/catalog/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+
+]
```

Comment:
Chưa define phần tử bên trong list thì em viết ngắn gọn như sau nhé
```suggestion
urlpatterns = []

```